db/migrate/20210928161912_migrate_deprecated_html_setting_type.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class MigrateDeprecatedHtmlSettingType < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      UPDATE site_settings
+      SET data_type = 1
+      WHERE data_type = 25
+    SQL
+  end
+
+  def down
+  end
```

Comment:
For consistency with other migrations, let's raise a irreversible migration error here.